theme/themes/pastanaga/elements/header.overrides

```diff
@@ -1,3 +1,85 @@
 /*******************************
          Theme Overrides
 *******************************/
+
+.ui.basic.segment.header-wrapper {
+  margin-bottom: 0;
+}
+
+.ui.basic.segment .header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-right: -@mini;
+  margin-left: -@mini;
+
+  & > * {
+    margin-right: @mini;
```

Comment:
@ksuess Why did you do this? I tend to like the logo sharp with the container (same at the end). 
<img width="847" alt="image" src="https://user-images.githubusercontent.com/486927/75369283-77ccd100-58c3-11ea-97d7-b1c30aa66488.png">

The rest is fine for me!

Reply:
It's a way to give items in a flex container identical margin.
  margin-right: -@mini;
  margin-left: -@mini;
  & > * {
    margin-right: @mini;
    margin-left: @mini;
  }
But in this case it's an obsolete left over of an earlier state where three items had to be distributed, not two (logo-nav-wrapper and tools-search-wrapper). I'll do that simpler.

Reply:
BTW the logo is in both cases exactly aligned, but has a padding in image itself.